litgpt/config.py

```diff
@@ -934,6 +945,66 @@ def norm_class(self) -> Type:
         gelu_approximate="tanh",
         intermediate_size=24576,
     ),
+    # https://huggingface.co/google/gemma-2-9b/blob/main/config.json
+    dict(
+        name="Gemma-2-9b",
+        hf_config=dict(org="google", name="gemma-2-9b"),
+        scale_embeddings=True,
+        # In Gemma attention scores are scaled not by `sqrt(head_size)` (16),
+        # but by `sqrt(n_emb // n_head)` = sqrt(3584 // 16) = 14.96
+        attention_scores_scalar=224,
```

Comment:
Apparently now it's 256.
https://huggingface.co/google/gemma-2-9b/blob/beb0c08e9eeb0548f3aca2ac870792825c357b7d/config.json#L24

Reply:
Ohhh interesting.

Btw I was trying to finetune the model yesterday but need QLoRA for that, and that seems to have issues for some reason I can't explain yet ... #1604

Reply:
Gemma2Config doesn't know about this new value, so it's better to keep `attention_scores_scalar` for 9b model even though it's the same value as `head_size`.